woocommerce/woocommerce-ios
# [IAP] Completed Upgrade view

Branched from https://github.com/woocommerce/woocommerce-ios/pull/10018 , which needs to be merged first.


## Description

We're adding the ability to purchase Woo Express plans using In App Purchase. This PR styles the `CompletedUpgradeView` which will appear when an upgrade has been successfully completed.

## Testing instructions
1. Change `UpgradesViewModel:117` to `.completed` state:
```
upgradeViewState = .completed
```
2. On a Free Trial store, tap on "Upgrade Now" > See the celebration screen > Tap on "Done" to dismiss the view.

## Screenshots

![Simulator Screen Recording - iPhone 11 Pro - 2023-06-21 at 12 34 32](https://github.com/woocommerce/woocommerce-ios/assets/3812076/a90daef3-9c1f-4276-a0eb-06700e90ba5b)
